package/gluon-web-private-wifi/luasrc/lib/gluon/config-mode/model/admin/privatewifi.lua

```diff
@@ -45,6 +45,13 @@ if platform.device_supports_mfp(uci) then
 end
 mfp.default = uci:get('wireless', primary_iface, 'ieee80211w') or "0"
 
+local ieee80211r = s:option(Flag, "80211r", translate("Fast BSSID transition"))
+ieee80211r.default = uci:get('wireless', primary_iface, 'ieee80211r') or false
+
+local mobility_domain = s:option(Value, "mobility_domain", "Mobility Domain")
+mobility_domain:depends(ieee80211r, true)
+mobility_domain = uci:get('wireless', primary_iface, 'mobility_domain') or '4f57'
```

Comment:
Will be dropped, since it is an advanced feature and we are compatible with OpenWrt anyhow based on the matching ESSID.